src/found_license.rs

```diff
@@ -258,8 +258,18 @@ impl Licensed for Package {
             .as_ref()
             .and_then(|license| license.parse::<License>().ok())
             .or_else(|| {
-                self.license_file()
-                    .map(|p| License::File(p.into_std_path_buf()))
+                self.license_file().map(|license_file| {
+                    let license_file = if let Some(stripped_license_file) = home::cargo_home()
```

Comment:
```suggestion
                self.license_file().map(|license_file| {
                    // If the license file starts with the cargo_home path, strip it and replace with the ENV var $CARGO_HOME.
                    // This makes licenses comparable across machines. 
                    let license_file = if let Some(stripped_license_file) = home::cargo_home()
```

Reply:
I have amended the first commit with the suggested comment.
Did you mean to disable the clippy lint just for this instance or in general?
I have currently done the former in the last commit.

Reply:
You did it as I was thinking 👍